pymc3/distributions/continuous.py

```diff
@@ -2586,10 +2587,45 @@ class ChiSquared(Gamma):
     nu: int
```

Comment:
the old documentation is wrong, nu can be a float

Reply:
fixed! thanks for pointing this out